tests/garage/tf/policies/test_qf_derived_policy.py

```diff
@@ -0,0 +1,53 @@
+"""
+This script creates a unittest that tests Gaussian policies in
+garage.tf.policies.
+"""
+from unittest import mock
+
+import numpy as np
+import tensorflow as tf
+
+from garage.misc.overrides import overrides
+from garage.spaces import Box
+from garage.spaces import Discrete
+from garage.tf.core.mlp import mlp
+from garage.tf.policies import QfDerivedPolicy
+from tests.fixtures import TfGraphTestCase
+
+
+class TestQfDerivedPolicy(TfGraphTestCase):
+    @overrides
+    def setUp(self):
+        self.observation_shape = (4, )
+        self.action_shape = 2
+        self.env = mock.Mock()
```

Comment:
What's the difference with mock.Mock() here and tests.fixtures.DummyEnv? I think DummyEnv is more convenient to use.